pywbem_mock/_mainprovider.py

```diff
@@ -1009,24 +1092,30 @@ def CreateClass(self, **params):
     #
     ####################################################################
 
-    def ModifyClass(self, **params):
-        # pylint: disable=invalid-name,unused-argument
+    def ModifyClass(self, namespace, ModifiedClass):
+        # pylint: disable=unused-argument
         """
         Currently not implemented
-        Implements a mock server responder for
+        Implements a  CIM repository server responder for
         :meth:`~pywbem.WBEMConnection.MmodifyClass`
 
-        Modifies a new class in the repository.  Nothing is returned.
-        Emulates WBEMConnection.CreateClass(...))
+        Modifies a class in the datastore.  Nothing is returned.
 
-        if the class repository for this namespace does not
-        exist, this method creates it.
+        Parameters:
+
+          namespace (:term:`string`):
+            Name of the namespace containing the class that will be modified
+            (case independent).
+
+            Leading and trailing slash characters will be stripped. The lexical
+            case will be preserved.
+
+          TODO
 
         Raises:
 
             CIMError: CIM_ERR_NOT_SUPPORTED
         """
-        namespace = self.namespace
 
         self.validate_namespace(namespace)
 
```

Comment:
Add a TODO for the implementation.

Reply:
self.namespace completely removed from code DONE